performance-evaluation/setup-performance-evaluation.py

```diff
@@ -66,6 +65,37 @@ def is_valid_date_to_append(date_to_append):
 		return False
 	return True
 
+def is_valid_title(title):
```

Comment:
You're basically just checking `if title` here. Since `None` and `''` will both return `False` on that check.
Still, I'm not entirely against it but bear in mind that you could delete this function entirely and just do `if not title: ...` below in L79.